### ISSUE TITLE
Pulumi Website : View Stack : Filter on "Function" does not work?

### ISSUE THREAD

**Issue by StefH** [pulumi/pulumi#4248]
# Overview
I've a Azure Function defined:
![image](https://user-images.githubusercontent.com/249938/78008346-d8768000-733f-11ea-875a-21925946ce7b.png)

# Filter op Function
However, when filtering on "Function", nothing is returned?
![image](https://user-images.githubusercontent.com/249938/78008390-e3c9ab80-733f-11ea-9f89-23c763033720.png)

**Comment by leezen** [comment]
This looks like a bug. I think if you filter by "Containers" you'll see the resource. This is because right now, the entire `azure:appservice` type space is mapped to `Container`

I'll add this to the backlog.

**Comment by StefH** [comment]
Correct:
![image](https://user-images.githubusercontent.com/249938/78103960-7c1e6980-73ee-11ea-8e1f-7821f2e29fd6.png)


However, the ASP is also returned here...

**Comment by leezen** [comment]
Yup -- that's definitely not the intended behavior. Currently, everything under the `azure:appservice` namespace will show up with the `Containers` filter (which is a bug)
